tests/oid_tests.rs

```diff
@@ -6,6 +6,15 @@ fn test_oid_value() {
     );
 }
 
+#[test]
+fn test_oid_with_uuid() {
+    assert_eq!(
+        asn1::oid!(2, 25, 223663413560230117710484359924050447509),
+        asn1::ObjectIdentifier::from_string("2.25.223663413560230117710484359924050447509")
+            .unwrap()
```

Comment:
Can we assert something specific about these? (Even if it's just their to_string())

Reply:
Added coverage and additionally asserted string equality for the OIDs commented on above.